Makefile

```diff
@@ -30,10 +30,10 @@ endef
 .PHONY: all
 all: Makefile .docker-image conf
 	if [ -e ./mbed_cloud_dev_credentials.c ]; then \
-		cp ./mbed_cloud_dev_credentials.c  ${POKY}/meta-pelion-edge/recipes-wigwag/mbed-edge-core/files/; \
+		cp ./mbed_cloud_dev_credentials.c  ${POKY}/meta-mbed-edge/recipes-connectivity/mbed-edge-core/files/; \
```

Comment:
Is the meta-mbed-edge coming from the GitHub repo name?

Reply:
We have two repos; 
- https://github.com/pelioniot/meta-pelion-edge
- https://github.com/PelionIoT/meta-mbed-edge
Do we have one repo too many?

Reply:
Yes,  Plan of record:

1. Accept this PR and establish a stable build that a) allows for CI/CD to work b) allows other dev's to develop the other features
2. Build out a PR that establishes meta-pelion-edge/{meta-pe-base,meta-pe-systems management,meta-pe-container-orchestration}
3. Move the contents of this meta into meta-pe-base